src/Illuminate/Session/DatabaseSessionHandler.php

```diff
@@ -198,6 +198,7 @@ protected function addUserInformation(&$payload)
     {
         if ($this->container->bound(Guard::class)) {
             $payload['user_id'] = $this->userId();
+            $payload['user_type'] = $this->userModel();
```

Comment:
Forgive me if I'm wrong but surely this would either return null or a string (getMorphClass returns a string) yet your database field for the user_type is foreignId (unsignedBigInteger)?

Reply:
sorry the field should be string